releasenotes/notes/fix-bedrock-input-tokens-unified-calculation-b2c3d4e5f6g7h8i9.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    LLM Observability: This fix resolves an issue where bedrock prompt caching caused input token counts to be the 
```

Comment:
You might rephrase this as "input token counts were miscounted during bedrock prompt caching". The way it's currently phrased says that the prompt caching is the cause of the issue. Since prompt caching is a third party feature (I think), it wouldn't really be the cause of a bug in our library.